src/Components/Contact_US/Contact_US.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
-import styles from "./Contact.module.css";
+import styles from "./Contact_US.module.css";
 
-const Contact = () => {
+const Contact_US = () => {
```

Comment:
Never mix two different types of naming conventions, either make it snake-case or PascalCase